src/commcare_cloud/ansible/roles/commcarehq/tasks/management_commands.yml

```diff
@@ -11,3 +11,7 @@
     - env_vars:
         TMPDIR: '{{ encrypted_tmp }}'
         PROMETHEUS_MULTIPROC_DIR: "{% if prometheus_monitoring_enabled|default(False) %}{{ metrics_home }}{% endif %}"
+        DD_TRACE_ENABLED: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
+        DD_TRACE_DJANGO_ENABLED: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
+        DD_TRACE_DEBUG: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
+        DD_INSTRUMENTATION_TELEMETRY_ENABLED: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
```

Comment:
Answering my own question, it looks like `env_vars` is our own thing, and we do not set variables with empty values ([related](https://github.com/dimagi/commcare-cloud/pull/6404/commits/e10fef44cff46a5103642869379a4da7b52b2874)).

Reply:
> Answering my own question, it looks like `env_vars` is our own thing, and we do not set variables with empty values ([related](https://github.com/dimagi/commcare-cloud/pull/6404/commits/e10fef44cff46a5103642869379a4da7b52b2874)).

That's right, and it's a smart way to enable users of a role/task to specify that the envvar shouldn't be set at all.

The awkward thing with the `DD_TRACE_*` variables is that they [default to being set (`True`) when they are not set](https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_TRACE_ENABLED). It would've made more sense if either they defaulted to false, or they default to true but with the inverse variable `DD_TRACE_DISABLE` to override this.